### DISCUSSION TITLE
redirect to privacy respecting sites

### DISCUSSION THREAD

**Discussion by vayaasco** [TrianguloY/URLCheck#203] [Ideas]
It will be perfect to integrate a redirection from YouTube to invidious, twitter yo nitter, reddit yo teddit... like does UntrackMe but completely integrated

**Comment by TrianguloY** [comment]
It is possible to use the Patterns checker module to do it! The configuration is a bit advanced, but some users created an external site with ready to use ones.

Find all info here: https://github.com/TrianguloY/UrlChecker/wiki/Custom-patterns
